staging/src/k8s.io/pod-security-admission/policy/check_allowPrivilegeEscalation_test.go

```diff
@@ -40,17 +45,56 @@ func TestAllowPrivilegeEscalation_1_25(t *testing.T) {
 					{Name: "c", SecurityContext: &corev1.SecurityContext{AllowPrivilegeEscalation: utilpointer.Bool(true)}},
 					{Name: "d", SecurityContext: &corev1.SecurityContext{AllowPrivilegeEscalation: utilpointer.Bool(false)}},
 				}}},
+			opts: options{
+				withFieldErrors: false,
+			},
```

Comment:
in reviewing the tests, it would help if the diff was as minimal as possible... `withFieldErrors: false,` is the default, so you *should* be able to omit `opts` from the unchanged testcases (suggestion applies everywhere)